register/models.py

```diff
@@ -104,7 +106,9 @@ def cancel(self):
             self.status = APP_CANCELLED
             self.save()
             m = MailListManager()
-            m.remove_recipient_from_list(self.sendgrid_id, m.WINTER_17_LIST_ID)
+            m.remove_recipient_from_list(self.sendgrid_id, m.W17_GENERAL_LIST_ID)
+            if self.scholarship:
+                m.remove_recipient_from_list(self.sendgrid_id, m.W17_TRAVEL_REIMB_LIST_ID)
```

Comment:
No error, if the user didn't belong to the list then removing her makes no change to sendgrid's side, but it doesn't cause any error. Just tested it (manually).